monkey/infection_monkey/island_api_client/http_island_api_client.py

```diff
@@ -201,6 +189,18 @@ def _serialize_events(self, events: Sequence[AbstractAgentEvent]) -> JSONSeriali
 
         return serialized_events
 
+    @handle_island_errors
+    @convert_json_error_to_island_api_error
+    def get_agent_signals(self, agent_id: str) -> Optional[datetime]:
+        url = f"{self._api_url}/agent-signals/{agent_id}"
+        response = requests.get(  # noqa: DUO123
+            url,
+            verify=False,
+            timeout=SHORT_REQUEST_TIMEOUT,
+        )
+        response.raise_for_status()
+        return response.json()["terminate"]
```

Comment:
```suggestion
        return response.json()["terminate_time"]
```

Reply:
I will change it in the final task.

![image](https://user-images.githubusercontent.com/15820737/191976375-2406c727-c538-4e85-9874-05c890c55825.png)